src/components/ContactForm/ContactForm.tsx

```diff
@@ -18,13 +20,13 @@ export default function ContactForm({
   showSuccessMsg: () => void
 }) {
   const { runAsync, loading } = useAsyncAction()
-
   const handleSendEmail = (values: ContactFormValues) => {
     runAsync(async () => {
       try {
         // Handle submit actions
         // console.log('Handle submit', values)
-        showSuccessMsg()
+        addSubscriberToMailerLite(values.senderEmail, contactGroup),
```

Comment:
**nit:** this also should be called in `action` from your PR with creating contact request, let's merge first, then we will adjust it